wbooks/src/app/screens/Home/index.tsx

```diff
@@ -1,17 +1,50 @@
-import React from 'react';
-import { ScrollView, Text, Button } from 'react-native';
-import { useNavigation } from '@react-navigation/core';
+import React, { useState, useEffect } from 'react';
+import { View, Text, TouchableOpacity, FlatList, Animated } from 'react-native';
+import { useDispatch } from 'react-redux';
+import bookAction from '@redux/books/actions';
 import { routesName } from '@constants/routesName';
+import noneBook from '@assets/books/noneBook.png';
+import { BOOKS_MOCK } from '@constants/mockBooks';
 
 import styles from './styles';
 
-function HomeScreen() {
-  const navigation = useNavigation();
+function HomeScreen({ navigation }: any) {
+  const [books, setBooks] = useState(BOOKS_MOCK);
+  const [animation] = useState(new Animated.Value(0));
+  const dispatch = useDispatch();
+  const keyExtractor = (item: any, index: number) => index.toString();
+
+  useEffect(() => {
+    setBooks(BOOKS_MOCK);
+    Animated.timing(animation, {
+      toValue: 1,
+      duration: 2000,
+      useNativeDriver: false
+    }).start();
+  }, [animation]);
+
+  const renderItem = ({ item }: any) => (
+    <TouchableOpacity
+      style={styles.containerList}
+      onPress={() => {
+        dispatch(bookAction.detailBookAction(item));
```

Comment:
Let's remove this action and use [params](https://reactnavigation.org/docs/params/) to pass the book around. You can leave the action in the actions.ts so we can work on that when we get to the redux cards